astropy/coordinates/tests/test_solar_system.py

```diff
@@ -437,9 +438,14 @@ def test_file_ephemeris_wrong_input():
     # Try loading a non-existing file:
     with pytest.raises(ValueError):
         get_body('earth', time, ephemeris='/path/to/nonexisting/file.bsp')
+
+    # NOTE: This test currently leaves the file open (ResourceWarning).
+    # To fix this issue, an upstream fix is required in jplephem
+    # package.
     # Try loading a file that does exist, but is not an ephemeris file:
-    with pytest.raises(ValueError):
-        get_body('earth', time, ephemeris=__file__)
+    with pytest.warns(ResourceWarning):
+        with pytest.raises(ValueError):
+            get_body('earth', time, ephemeris=__file__)
```

Comment:
I think comma works too.
```suggestion
    with pytest.warns(ResourceWarning), pytest.raises(ValueError):
        get_body('earth', time, ephemeris=__file__)
```

Reply:
It does, adopted.